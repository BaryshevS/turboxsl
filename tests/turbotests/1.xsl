<?xml version="1.0" encoding="UTF-8"?>
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:ltr="LTR">
<xsl:template match="test">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	</head>
	<body>
		<ol>
		<li><xsl:value-of select="@username"/></li>
		<li><xsl:value-of select="@fullname"/></li>
		<li><xsl:value-of select="@username | @fullname"/></li>
		<li><xsl:value-of select="@fullname | @username"/></li>
		<li><xsl:value-of select="sub_test/@username | @fullname"/></li>
		<li><xsl:value-of select="@username | sub_test/@fullname"/></li>
		<li><xsl:value-of select="sub_test/@username | sub_test/@fullname"/></li>
		<li><xsl:value-of select="sub_test/@fullname | sub_test/@username"/></li>
		<li><xsl:value-of select="sub_test/@username | sub_test2/@username"/></li>
		<li><xsl:value-of select="sub_test2/@fullname|sub_test/@fullname|sub_test/../@fullname"/></li>
		<li><xsl:value-of select="sub_test2/@fullname|sub_test/@fullname"/></li>
		</ol>
		<xsl:apply-templates/>
	</body>
</xsl:template>

</xsl:stylesheet>
