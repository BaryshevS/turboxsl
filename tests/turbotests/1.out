<?xml version="1.0"?>
<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/></head><body><ol><li>user</li><li>Oo</li><li>userOo</li><li>Oouser</li><li>sub_userOo</li><li>usersub_Oo</li><li>sub_usersub_Oo</li><li>sub_Oosub_user</li><li>sub_usersub_user2</li><li>sub_Oo2sub_OoOo</li><li>sub_Oo2sub_Oo</li></ol>
	
	
	123
</body>